### PR TITLE
Add docs for stock-market-activity labs

### PR DESCRIPTION
**Pull request by JakeSCahill**
Fixes https://github.com/redpanda-data/documentation-private/issues/1492

Previews:

- [Node.js](https://deploy-preview-37--redpanda-labs-preview.netlify.app/redpanda-labs/clients/stock-market-activity-nodejs/)
- [Python](https://deploy-preview-37--redpanda-labs-preview.netlify.app/redpanda-labs/clients/stock-market-activity-python/)

### CODE REVIEW

File: clients/stock-market-activity/nodejs/README.adoc [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+| Specify the path to the CSV file to be processed. Defaults to `../data/market_activity.csv`.
+
+| `-t`, `--topic`
+| Specify the Kafka topic to which events will be published. Defaults to the name of the CSV file (without its extension).
+
+| `-b`, `--broker`, `--brokers`
+| Comma-separated list of the host and port for each Kafka broker. Defaults to `localhost:9092`.
+
+| `-d`, `--date`
+| Specify the column in the CSV file that contains date information, which will be converted and manipulated based on the script settings.
```
Comment (kbatuigas) [inline clients/stock-market-activity/nodejs/README.adoc:134]: Would it make sense to mention somewhere how the dates get manipulated by default? And then readers can look in the script to see the implementation / maybe we add a comment there so they can find it easily

**Review by kbatuigas — COMMENTED** [review]
Looks really good so far!

This is a general RP Labs question - do we expect readers to look at the scripts or application code through the repo that they've cloned? In the deploy preview I tried to navigate to it on GitHub (via the "Source Code" link on the left nav) and didn't know where to go from there. (Sorry if you had already previously covered this in a demo Jake and I just forgot)

**Comment by JakeSCahill** [comment]
> This is a general RP Labs question - do we expect readers to look at the scripts or application code through the repo that they've cloned? In the deploy preview I tried to navigate to it on GitHub (via the "Source Code" link on the left nav) and didn't know where to go from there. (Sorry if you had already previously covered this in a demo Jake and I just forgot)

Good point, thanks! The link currently goes to the location of the docs, but we could adjust this to open the the code one directory up from the docs.

File: clients/stock-market-activity/nodejs/README.adoc (hunk 1 above)
Comment (JakeSCahill) [reply clients/stock-market-activity/nodejs/README.adoc]: Improved the description and added code comments 👍

**Review by kbatuigas — APPROVED** [review]

**Pull request merged**